api/src/main/java/org/openmrs/module/fhir2/apiext/dao/impl/FhirMedicationAdministrationNoteDaoImpl.java

```diff
@@ -0,0 +1,10 @@
+package org.openmrs.module.fhir2.apiext.dao.impl;
+
+import org.openmrs.module.fhir2.api.dao.impl.BaseFhirDao;
+import org.openmrs.module.fhir2.apiext.dao.FhirMedicationAdministrationNoteDao;
+import org.openmrs.module.ipd.api.model.MedicationAdministrationNote;
+import org.springframework.stereotype.Component;
+
+@Component
+public class FhirMedicationAdministrationNoteDaoImpl extends BaseFhirDao<MedicationAdministrationNote> implements FhirMedicationAdministrationNoteDao {
+}
```

Comment:
Since we have overridden the get method in DAO, can we have a implementation as well which return super.get(uuid) ? or we can we remove this impl since it is empty

Reply:
@kavitha-sundararajan  Added Implementation which will return super.get(uuid)  ! Will not be able to remove this Impl as we will not able to autowire it without this @Component